shared/src/business/useCases/courtIssuedDocument/serveCourtIssuedDocumentInteractor.ts

```diff
@@ -233,3 +215,28 @@ export const serveCourtIssuedDocumentInteractor = async (
     userId: user.userId,
   });
 };
+
```

Comment:
Nit: Is it worth extracting this to a utility method since we have it defined locally in 2 diff files now?

Reply:
that's a good point